src/Altinn.Notifications.Persistence/Repository/EmailNotificationRepository.cs

```diff
@@ -75,11 +75,12 @@ await using (NpgsqlDataReader reader = await pgcom.ExecuteReaderAsync())
     }
 
     /// <inheritdoc/>
-    public async Task SetResultStatus(Guid notificationId, EmailNotificationResultType status)
+    public async Task UpdateSendStatus(Guid notificationId, EmailNotificationResultType status, string? operationId)
```

Comment:
what if we set a default value null for operationId? making it obsolete to change existing use of the method ?